Facepunch.Steamworks/SteamFriends.cs

```diff
@@ -298,5 +298,22 @@ public static bool ListenForFriendsMessages
 			}
 		}
 
+		public static async Task<bool> IsFollowing(SteamId steamID)
+		{
+			var r = await Internal.IsFollowing(steamID);
+			return r.Value.IsFollowing;
+		}
+
+		public static async Task<int> GetFollowerCount(SteamId steamID)
+		{
+			var r = await Internal.GetFollowerCount(steamID);
+			return r.Value.Count;
+		}
+
+		public static async Task<ulong[]> EnumerateFollowingList(uint StartIndex)
```

Comment:
This isn't too good - maybe convert them to steamids, call it GetFollowingList. Is there much reason for a startindex?

Reply:
Done.